### PR TITLE
Document the new HEAD call for annotations

### PR DESCRIPTION
**Pull request by curtis1000**
Refs WEB-2213, WEB-4852

### CODE REVIEW

**Comment by shadowhand** [comment]
:+1:

File: source/methods/annotations.md.erb [hunk 1]
```diff
@@ -147,3 +147,39 @@ Annotations can be deleted by managers and supervisors who can supervise the rel
 Key | Description
 --- | -----------
 <% param "id" do %>ID of the annotation to be deleted.<% end %>
+
+## Head Call
```
Comment (bvisness) [inline source/methods/annotations.md.erb]: Nitpick: might as well make HEAD all caps here.

**Comment by bvisness** [comment]
👍 , should I merge and deploy it? @curtis1000

**Comment by curtis1000** [comment]
Please, the endpoint is released now, thanks!

**Pull request merged**